qgis-projects/README.md

```diff
@@ -0,0 +1,141 @@
+- Role: Primary Custodian
+- Description: Responsible for management of project wide geodata and GIS projects
+- Name: Charles Dixon-Paver
+- Mail: charles@kartoza.com
+- GitHub: @zacharlie
+
+---
+
+- Custodian: Lead
+- Role: Project Leader
+- Description: Responsible for decision, especially for making changes to project data structures
+- Name: Ricardo Garcia Silva
+- Mail: ricardo@kartoza.com
+- GitHub: @ricardogsilva
+
+---
+
+## Map Projects
+
+When creating a new map project, please ensure that this section is maintained with the relevant details.
+
+The naming convention for maps and associated files is to use hyphen-separated, or *kebab-case*, styling.
+
+> Map projects should be saved and stored as *qgs* documents (xml) **wherever possible** and not *qgz* documents (zipped archives, which are binary).
+
+The following elements outline the available projects within this repository.
+
+---
+
+* Project: dominode-topomaps.qgs
+* Custodian: Default
+* Description: Primary map project for toposheet production
+
+---
+
+## Geodata descriptions
+
+The following sections detail the relevant sources and file structures for geodata that are utilised within the projects in this directory.
+
+### Quickstart
+
+Download the relevant project data sources, usually indicated by a *./data* source in the following table. Remote sources should be available within map projects directly (assuming the relevant connections have been configured correctly within QGIS). Local data sources are stored within the repository.
+
+Once the data has been downloaded to the local system, create a symbolic link at ```qgis-projects/data``` which may redirect to the relevant filepath. Symbolic links can be relative or absolute.
+
+Linux and Mac users can create a symbolic link using the ```ln``` command, e.g. ```ln -s /home/user/data/project ./data```
+
+Windows users may use the ```mklink``` function from *cmd*, e.g. ```mklink /D ../../../data ./data```.
+
+### Project data
+
+Adding data should be done directly by primary project custodians wherever possible. Adding of features or layers to existing data sources (remote/ gpkg etc) should by distributed in the same manner as the original data source (republished) or have their source details modified in the following section.
+
+---
+
+- File: ./data/dominode_data.gpkg
+- Custodian: Default
+- Description: Local copy of all data directly copied from remote staging database. It is recommended that this is copied directly from the Dominode staging environment database.
+- Source: Connect to the database, create a local geopackage named dominode_data.gpkg and select all features, then drag and drop them onto the gpkg within the QGIS Browser. Contact the custodian for database connection details.
+ 
+---
+
+- File: ./data/auxillary_data.gpkg
+- Custodian: Default
+- Description: Additional data required but not present in core database. This data should be accounted for in the remote data source later on.
```

Comment:
Is there any particular reason why this data is not on the above geopackage file?
If not it would be better to consolidate it into a single file, to make it easier to use